pkg/controller/control.go

```diff
@@ -283,14 +284,14 @@ func (c *Controller) startExpansion(size int64) (err error) {
 	if c.hasWOReplica() {
 		return fmt.Errorf("cannot do expansion since there is WO(rebuilding) replica")
 	}
+	if size%diskutil.VolumeSectorSize != 0 {
+		return fmt.Errorf("controller expanding size %v is not multiple for volume sector size %v", size, diskutil.VolumeSectorSize)
```

Comment:
`"failed to expand volume size %v, because it is not multiple of volume sector size %v"`